diary-alexander-jaffray/modernized/31-ch-15-scotland.adoc

```diff
@@ -331,54 +346,57 @@ And all the creation is upheld by his Word and power,
-that you may present them to God as his children, and all your servants and tenants,
+that you may present them to God _as his children,_ and all your servants and tenants,
 in the wisdom of God.
-You must answer the Truth in them all, in truth, holiness, righteousness, and justice,
+_You must answer the Truth in them all,_ in truth, holiness, righteousness, and justice,
 and walking humbly before God.
-You will always feel his presence to assist,
-and enable you to perform whatsoever he requires of you; so that whatever you do,
+_You will always feel his presence to assist,
+and enable you to perform whatsoever he requires of you;_ so that whatever you do,
 it may be to the honour and glory of God.
-And do not look at the outward presence of your husband; but look at the Lord,
-and serve Him with a joyful heart, mind, soul,
+And _do not look at the outward presence of your husband;_ but look at the Lord,
+and serve Him _with a joyful heart,_ mind, soul,
 and spirit all the days you live upon the earth.
 
 From him, who had a great love and respect for your dear husband,
 for his work and service in the Lord, who is content in the will of God,
-and all things that he does:--and so must you be.
+and all things that he does:--__and so must you be.__
 And so, the Lord God Almighty,
 settle and establish you and yours upon the heavenly Rock and Foundation; that,
 as your children grow in years, they may grow in grace, and so in favour with the Lord.
 Amen!
 
+[.signed-section-signature]
 George Fox.
 
 [.postscript]
 ====
 
 Postscript.--I know your husband has left a good savour behind him,
-so I desire you may do the same.
+so I desire _you may do the same._
 
 ====
 
+--
+
 The following faithful delineation of the character of "`this
 worthy young man of God,`" as William Penn styles him,
-may be fresh to many readers; but is worthy the repeated attentive perusal of those,
+may be fresh to many readers; but is worthy the _repeated_ attentive perusal of those,
 who claim connection with the Society of Friends, especially among the younger classes.
 To adopt the language and motives of the same writer, William Penn,
-on the like occasion,--"`For their example and encouragement,`" is this account given,
+on the like occasion,--"`For _their_ example and encouragement,`" is this account given,
 "`who have or hereafter may receive the eternal Truth,
-as well as for a testimony to the power and goodness
-of God in raising him up to his church.`"
+as well as __for a testimony to the power and goodness
+of God in raising him up to his church.__`"
 It is prepared from documents,
 put forth by those contemporaries of Robert Barclay who knew him well,
-and appears in the pages of A Short Account of His Life and Writings.
+and appears in the pages of [.book-title]#A Short Account of His Life and Writings.#
 
 He was distinguished by strong mental powers, particularly by great penetration,
```

Comment:
So, from here to the end of the chapter is a quote from the book A Short Account of the Life and Writings of Robert Barclay, by John G. Bevan. So, I think these three paragraphs should be a block quote. 👍🏻